backend/tests/events.test.ts

```diff
@@ -92,3 +93,127 @@ describe ("Testing DELETE event",() => {
+
+    const events = await request(app).get("/events");
+    const eventid = events.body[0].id;
+    const response = await request(app).get("/events/" + eventid + "/attendees");
+    expect(response.status).toBe(200);
+  })
+})
+
+  test("Get attendees for invalid event", async () => {
+
+    const eventid = -1;
+    const response = await request(app).get("/events/" + eventid + "/attendees");
+    expect(response.body.length).toBe(0);
+  })
+
+  describe ("Testing POST/events/:eventid/:attendeeid", () => {
+    test("Add attendee for existing event", async () => {
+
+      const events = await request(app).get("/events");
+      const users = await request(app).get("/users");
+      const eventID = events.body[1].id;
+      const attendeeid = users.body[1].id;
+      const response = await request(app).post("/events/" + eventID + "/"+ attendeeid);
+      expect(response.status).toBe(200);
+    })
+  })
+
+  describe("Testing DELETE /events/:eventid/attendees/:attendeeid", () => {
+    test("Delete an attendee", async () => {
+      const events = await request(app).get("/events");
+      const attendees = await request(app).get("/users");
+      const eventid = events.body[1].id;
+      const attendeeid = attendees.body[0].id;
+      const response = await request(app).delete("/events/" + eventid + "/attendees/" + attendeeid);
+      expect(response.status).toBe(200);
+    });
+
+    test("Delete an invalid attendee", async () => {
+      const events = await request(app).get("/events");
+      const eventid = events.body[1].id;
+      const attendeeid = events.body[0].userID;
+
+      const response = await request(app).delete("/events/" + eventid + "/attendees/" + attendeeid);
+      expect(response.status).toBe(500);
+    });
+  });
+
+  describe("Testing PATCH /events/:eventid/status/:status", () => {
+    test("Update event status to active", async () => {
+      const events = await request(app).get("/events");
+      const eventid = events.body[0].id;
+      const status = "ACTIVE";
+      const response = await request(app).patch(`/events/${eventid}/status/${status}`)
+      expect(response.status).toBe(200);
+    });
+    
+    test("Event status invalid", async () => {
+      const events = await request(app).get("/events");
+      const eventid = events.body[0].id;
+      const status = null;
+      const response = await request(app).patch("/events/" + eventid + "/status/" + status);
+      expect(response.status).toBe(500);
+    });
+  });
+
+  describe("Testing PATCH /events/:eventid/owner/:ownerid", () => {
+    test("Change current owner", async () => {
+      const events = await request(app).get("/events");
+      const eventid = events.body[1].id;
+      const users = await request(app).get("/users");
+      const ownerid = users.body[0].id;
+      const response = await request(app).patch("/events/" + eventid + "/owner/" + ownerid); 
+      expect(response.status).toBe(200);
```

Comment:
Again, check if the updated record now reflects the change you want - that is how you test that your function actually does what it is expected to do.